src/hotspot/share/gc/shared/space.cpp

```diff
@@ -146,52 +140,10 @@ void DirtyCardToOopClosure::do_MemRegion(MemRegion mr) {
   _min_done = bottom;
 }
 
-HeapWord* ContiguousSpaceDCTOC::get_actual_top(HeapWord* top,
-                                               HeapWord* top_obj) {
-  if (top_obj != NULL && top_obj < (_sp->toContiguousSpace())->top()) {
-    if (cast_to_oop(top_obj)->is_objArray() || cast_to_oop(top_obj)->is_typeArray()) {
-      // An arrayOop is starting on the dirty card - since we do exact
-      // store checks for objArrays we are done.
-    } else {
-      // Otherwise, it is possible that the object starting on the dirty
-      // card spans the entire card, and that the store happened on a
-      // later card.  Figure out where the object ends.
-      assert(_sp->block_size(top_obj) == cast_to_oop(top_obj)->size(),
-        "Block size and object size mismatch");
-      top = top_obj + cast_to_oop(top_obj)->size();
-    }
-  } else {
-    top = (_sp->toContiguousSpace())->top();
-  }
-  return top;
-}
-
-void ContiguousSpaceDCTOC::walk_mem_region(MemRegion mr,
-                                           HeapWord* bottom,
-                                           HeapWord* top) {
-  // Note that this assumption won't hold if we have a concurrent
-  // collector in this space, which may have freed up objects after
-  // they were dirtied and before the stop-the-world GC that is
-  // examining cards here.
-  assert(bottom < top, "ought to be at least one obj on a dirty card.");
-
-  if (_boundary != NULL) {
-    // We have a boundary outside of which we don't want to look
-    // at objects, so create a filtering closure around the
-    // oop closure before walking the region.
-    FilteringClosure filter(_boundary, _cl);
-    walk_mem_region_with_cl(mr, bottom, top, &filter);
-  } else {
-    // No boundary, simply walk the heap with the oop closure.
-    walk_mem_region_with_cl(mr, bottom, top, _cl);
-  }
-
-}
-
 // We must replicate this so that the static type of "FilteringClosure"
 // (see above) is apparent at the oop_iterate calls.
-#define ContiguousSpaceDCTOC__walk_mem_region_with_cl_DEFN(ClosureType) \
-void ContiguousSpaceDCTOC::walk_mem_region_with_cl(MemRegion mr,        \
+#define DirtyCardToOopClosure__walk_mem_region_with_cl_DEFN(ClosureType) \
+void DirtyCardToOopClosure::walk_mem_region_with_cl(MemRegion mr,        \
                                                    HeapWord* bottom,    \
                                                    HeapWord* top,       \
                                                    ClosureType* cl) {   \
```

Comment:
Parameter indentation is off now with the change, also the macro continuation `\` characters do not line up any more.